pegasus/sites.v3/code.org/public/teacher-dashboard/section_assessments.haml

```diff
@@ -11,7 +11,7 @@ content-type: text/ng-template
 
 %div{'ng-show' => 'assessmentsLoaded'}
   = I18n.t('dashboard_assessments_view')
-  %select{"ng-model" => "script_id", "ng-options"=>"script.id as script.name group by script.category for script in script_list | orderBy:['category_priority', 'category', 'position', 'name']", 'ng-change' => 'changeScript(script_id)'}
+  %select#uitest-course-dropdown{"ng-model" => "script_id", "ng-options"=>"script.id as script.name group by script.category for script in script_list | orderBy:['category_priority', 'category', 'position', 'name']", 'ng-change' => 'changeScript(script_id)'}
```

Comment:
Not sure why, but I generally prefer classes to ids for UI test tags like this.  Open to reasons ids are better here.